### PR TITLE
Tag LazyQuery.jl v0.1.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [bramtayl/LazyQuery.jl](https://github.com/bramtayl/LazyQuery.jl)
Release: [v0.1.0](https://github.com/bramtayl/LazyQuery.jl/releases/tag/v0.1.0)
Travis: [Travis Build Status](https://travis-ci.org/bramtayl/LazyQuery.jl/builds/252250714)
Diff: [vs v0.0.1](https://github.com/bramtayl/LazyQuery.jl/compare/e58692ea8e53e660ec67d18eb0f3d17feb780c1a...50bb94d41359fff22ddf51258496810075955590)
`requires` vs v0.0.1: 
```diff
--- 0.0.1/requires
+++ 0.1.0/requires
@@ -1,4 +1,6 @@
-julia 0.5
+LazyContext
 DataFrames
 MacroTools
-LazyContext 0.0.1 0.1.0
+Query
+NamedTuples
+DataFramesMeta
```
cc: @bramtayl

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds are up to date.

### CODE REVIEW

**Comment by tkelman** [comment]
Don't delete your julia requirement

**Comment by bramtayl** [comment]
oops

File: LazyQuery/versions/0.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,7 @@
+julia 0.6
+LazyContext
+DataFrames
+MacroTools
```
Comment (tkelman) [inline LazyQuery/versions/0.1.0/requires]: `@match` requires 0.3.3

are you using anything from LazyContext that didn't exist prior to 0.1.0, or 0.1.1?

**Comment by bramtayl** [comment]
Sorry for the noise just trying to get this right...

File: LazyQuery/versions/0.1.0/requires (hunk 1 above)
Comment (bramtayl) [inline LazyQuery/versions/0.1.0/requires]: yeah just the new constructor in 0.1.1. I've updated accordingly

**Pull request merged**